dataactcore/aws/s3UrlHandler.py

```diff
@@ -132,3 +132,14 @@ def get_file_urls(self, bucket_name, path):
                 urls[file_name] = url
 
         return urls
+
+    @staticmethod
```

Comment:
I don't think it's mandatory now to fix it but the naming should be relevant. If you can write up a ticket to ensure we at least are aware of it, that should suffice.